relayer-cli/src/conclude.rs

```diff
@@ -252,6 +248,16 @@ fn serialize_result(res: impl Serialize + std::fmt::Debug) -> serde_json::Value
     }
 }
 
+impl From<Output> for String {
```

Comment:
Unless there is a good reason for extracting this code out of `exit_with`, can we move it back there?
I am a bit wary of having a `From` instance whose behavior depends on a runtime condition (ie. whether JSON output is enabled or not). Maybe we can instead move this code into a method on `Output`?

Reply:
Done 👍 ...